.idea/codeStyles/Project.xml

```diff
@@ -0,0 +1,116 @@
+<component name="ProjectCodeStyleConfiguration">
```

Comment:
I don't remember adding this file, but it keeps popping up, so will leave it in the repo.